src/range.rs

```diff
@@ -227,6 +228,6 @@ mod tests {
 
         verifier.preprocess(&ck)?;
 
-        verifier.verify(&proof, &vk, &vec![BlsScalar::zero()])
+        verifier.verify(&proof, &vk, &alloc::vec![BlsScalar::zero()])
```

Comment:
I would prefer leaving `&vec!` without the full domain, as we did for the other crates that are using the same alloc's macro.